daemon/logger/journald/journald.go

```diff
@@ -48,10 +49,17 @@ func New(ctx logger.Context) (logger.Logger, error) {
 		name = name[1:]
 	}
 
+	// parse log tag
+	tag, err := loggerutils.ParseLogTag(ctx, "")
```

Comment:
I was wondering on this one too (as @MHBauer). For the other logging driver that supports the `tag`, the default is `docker.{{.ID}}` (the container short id). 

Although it duplicate the containerID by default in the log, shouldn't we have the same behavior accross all the logging driver that supports `tag`s ?